requirements.txt

```diff
@@ -4,7 +4,10 @@ netaddr>=0.7.11
 
 # AWS Provider
 boto3>=1.9.60
-opinel>=3.3.4,<4.0.0
+
+# Opinel
+requests>=2.4.0,<3.0.0
+iampoliciesgonewild>=1.0.6.2
```

Comment:
I didn't know we had this dependency. Hasn't been updated since 2016, I wonder if it supports new services? Could be worth checking where it's used and if the dependency can be removed.